source/topologies/rs-to-sharded.txt

```diff
@@ -0,0 +1,101 @@
+.. _c2c-rs-to-sharded:
+
+=======================================
+Sync a Replica Set to a Sharded Cluster
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+This page outlines the procedure to sync data between a replica set and a 
+sharded cluster using :program:`mongosync`. To perform a sync between two
+replica sets, see :ref:`c2c-quickstart`.
+
+Before You Begin
+----------------
+
+Before you begin your sync, define your source replica set, your 
+destination sharded cluster, and an administrative user that has permissions in 
+both clusters. You must also download ``mongosync``. To learn more, see 
+the :ref:`setup <c2c-quickstart-setup>` instructions.
+
+Steps 
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Connect the replica set and sharded cluster. 
+
+      ``mongosync`` must create an initial connection to the source and 
+      destination clusters before you can start to sync data.
+      Follow the :ref:`connection instructions <c2c-conn-top-level>` for your 
+      cluster architecture to connect to the :binary:`mongos` instance in your 
+      cluster.
+
+      For an example of the ``mongosync`` connection command, see 
+      :ref:`Initialize mongosync <c2c-initialize>`.
+
+   .. step:: Start synchronization.
+
+       The :ref:`start <c2c-api-start>` endpoint initiates data 
+       synchronization. 
+       
+       To sync from a replica set to a sharded cluster, set the 
+       ``sharding`` option for the ``start`` command to shard collections on 
+       the destination cluster. For more information, see 
+       :ref:`Sharding Parameters <c2c-api-start-sharding>`.
+
+       The ``sharding.shardingEntries`` array specifies the collections to shard.
+       Collections that are not listed in this array replicate as unsharded. For
+       more information, see :ref:`Shard Replica Sets <c2c-shard-replica-sets>`.
```

Comment:
[suggestion] we can probably use active voice here!

```suggestion
       Collections that you do not list in this array replicate as unsharded. For
       more information, see :ref:`Shard Replica Sets <c2c-shard-replica-sets>`.
```

Reply:
Fixing now!